browser/importer/brave_importer_p3a.cc

```diff
@@ -34,10 +34,10 @@ void RecordImporterP3A(importer::ImporterType type) {
   case importer::TYPE_IE:
     metric = ImporterSource::kIE;
     break;
+#endif
```

Comment:
we can keep this one as Windows - as it's legacy edge; and then (as described in previous comment) use a new type - like `importer::TYPE_CHROMIUM_EDGE`

Reply:
done